frontend/src/i18n/en/dashboard.ts

```
@@ -90,6 +90,30 @@ export default {
   'agent_sdk_desc': 'Powered by Claude Agent SDK, driving conversational AI assistants and automated workflows',
   'claude_code_compat_hint': 'Configuration is compatible with Claude Code environment variables and Coding Plan APIs.',
+  'ai_providers': 'AI Providers',
+  'ai_providers_desc': 'Manage AI providers for the assistant. Drag and drop or use arrows to set priority order. The system automatically falls back to the next provider if one fails.',
+  'add_provider': 'Add Provider',
+  'no_providers': 'No providers yet. Click "Add Provider" to begin.',
+  'active': 'ACTIVE',
+  'set_active': 'Set active',
+  'edit': 'Edit',
+  'delete': 'Delete',
+  'cannot_delete_active': 'Cannot delete the active provider',
+  'test_connection': 'Test connection',
```

Biome (2.4.16)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)